mava/utils/checkpointing.py

```diff
@@ -151,7 +152,7 @@ def restore_params(
         timestep: Optional[int] = None,
         restore_hstates: bool = False,
         THiddenState: Optional[Type] = None,  # noqa: N803
-    ) -> Tuple[Any, Union[HiddenStates, None]]:
+    ) -> Tuple[Any, Union[HiddenStates, SableHiddenStates, None]]:
```

Comment:
Too many hidden state types

```suggestion
    ) -> Tuple[Any, Optional[Any]]:
```